webknossos-datastore/app/com/scalableminds/webknossos/datastore/s3fs/S3FileSystemProvider.java

```diff
@@ -37,8 +37,8 @@
 /**
  * Spec:
  * <p>
- * URI: s3://[endpoint]/{bucket}/{key} If endpoint is missing, it's assumed to
- * be the default S3 endpoint (s3.amazonaws.com)
+ * URI: s3://[user@]{bucket}/{key}.
```

Comment:
Does the dot at the end mean end of statement or include a dot in the URL?

Reply:
good point, I removed it to avoid confusion. No dot in the URLs please